src/main/java/com/teamcautionrobotics/robot/Cargo.java

```diff
@@ -0,0 +1,55 @@
+
+package com.teamcautionrobotics.robot;
+
+import edu.wpi.first.wpilibj.*;
```

Comment:
Should this really be import * and not
```suggestion
import edu.wpi.first.wpilibj.VictorSP;
import edu.wpi.first.wpilibj.Solenoid;
```